en.html

```diff
@@ -0,0 +1,472 @@
+					<div class="navbar-brand">
+						<a href="index.html">
+							<img class="logo" src="img/logo.svg" alt="logo">
+							<img class="logo-alt" src="img/logo-alt.svg" alt="logo">
+						</a>
+					</div>
+
+					<div class="nav-collapse">
+						<span></span>
+					</div>
+				</div>
+
+				<ul class="main-nav nav navbar-nav navbar-right">
+					<li><a href="#home">Introduction</a></li>
+					<li><a href="#about">About Míčkoflusu</a></li>
+					<li><a href="#partners">Partners</a></li>
+					<li><a href="#gallery">Gallery</a></li>
+					<li><a href="#guide">Production guide</a></li>
+					<li><a href="#team">Who's behind it</a></li>
+					<li><a href="#contact">Contact</a></li>
+					<li><a href="index.html"> <span class="lang-sm lang-lbl" lang="cs"></span></a></li>
+				</ul>
+			</div>
+		</nav>
+
+		<div class="home-wrapper">
+			<div class="container">
+				<div class="row">
+
+					<div class="col-md-10 col-md-offset-1">
+						<div class="home-content">
+							<h1 class="white-text text-center">Míčkoflus</h1>
+							<p class="white-text text-center">Shoot them all!</p>
+							<a href="#about"><button class="white-btn">Get more information</button></a>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<div id="about" class="section md-padding">
+		<div class="container">
+			<div class="row">
+				<div class="section-header">
+					<h2 class="title">About Míčkoflusu</h2>
+					<p>It drives, shoots it and looks innocent. This is Míčkoflus &#8212;
+					robotics kit from Robotárna (department of <a target="_tab" href="https://helceletka.cz">
+					Youth Centre Helceletova Brno</a>). This kit was firstly introduce on <a href="">RoboticCamp 2018</a>.</p>
+				</div>
+
+				<div class="row row-eq-height">
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-tree"></i>
+							<h3>Unmistakable wooden design</h3>
+							<p>Strong, light, easy glueable. All thanks to wood.</p>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-cogs"></i>
+							<h3>5 independent motors</h3>
+							<p>Thanks to it, Míčkoflus can ride, aim and shoot. Encoders can be connected to the motors and controlled precisely.</p>
+						</div>
+					</div>
+
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-mobile"></i>
+							<h3>Control with smartphone</h3>
+							<p>In basic mode, Mičkoflus can be remote control from the smartphone through WiFi or Bluetooth.</p>
+						</div>
+					</div>
+				</div>
+
+				<div class="row row-eq-height">
+					<div class="col-md-4 xsm-padding">
+						<div class="about">
+							<i class="fa fa-child"></i>
+							<h3>Suitable for beginners but also advanced users</h3>
+							<p>Basic model can build everybody. But do you dare to build a larger ball tray or add a gyroscope and program your tower stabilization while driving?
```

Comment:
Everybody can build the basic model.
Dare je spíše negativní. Accept the challenge and build a larger...